src/components/toggle-button/assistant-departure-toggle-button.tsx

```diff
@@ -0,0 +1,80 @@
+import Link from 'next/link';
+import style from './assistant-departure-toggle-button.module.css';
+import { PageText, TranslatedString, useTranslation } from '@atb/translations';
+import { useState } from 'react';
+import { Typo } from '../typography';
+import { AnimatePresence, motion } from 'framer-motion';
+
+type AssistantDepartureTogglProp = {
+  href?: string;
+};
+
+type LinkProps = {
+  name: string;
+  title: TranslatedString;
+  link: string;
+};
+
+const links: LinkProps[] = [
+  {
+    name: 'assistant',
+    title: PageText.Assistant.title,
+    link: '/assistant',
+  },
+  {
+    name: 'departures',
+    title: PageText.Assistant.search.buttons.find.title,
```

Comment:
This could get its own translation entry, I think.

Maybe both of these (L21, L26) should be in ComponentText instead of PageText?

Reply:
Agreed! I didn't realize that ComponentText existed before you pointed it out with this comment. Will fix.